debian/control

```diff
@@ -82,6 +82,7 @@ Depends:
     pulseaudio,
     libpulse0,
     ${shlibs:Depends},
+    ${pulse:Depends},
```

Comment:
I don't know much about that. Where did you learn about `${pulse:Depends}` or where did you invent it?

Reply:
The latter - it is defined by the substvars file.